Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.19.10@sha256:83f9f840072d05ad4d90ce4ac7cb2427632d6b89d5ffc558f18f9577ec8188c0 as builder
+FROM golang:1.20.6@sha256:8e5a0067e6b387263a01d06b91ef1a983f90e9638564f6e25392fd2695f7ab6c as builder
```

Comment:
i think for capi 1.4.x it is on go 1.19.x (https://github.com/kubernetes-sigs/cluster-api/blob/release-1.4/Makefile#L26C13-L26C22) we should match that similar to the other providers as well

Reply:
It seems like they have updated it to 1.20.8 in this commit for CAPI 1.4.x [here](https://github.com/kubernetes-sigs/cluster-api/commit/6b7d2b2680354340433606462f208ac934963939). I'll update to 1.20.8 if that is fine with you ?

Reply:
sounds great!